get-additional-demos.sh

```diff
@@ -14,16 +14,22 @@ user=${V3IO_USERNAME}
 
 USAGE="\
 $SCRIPT:
-Retrieves latest-release demos from the mlrun/demos GitHub repo.
+Retrieves updated demos from the mlrun/demos GitHub repository.
 USAGE: ${SCRIPT} [OPTIONS]
 OPTIONS:
   -h|--help   -  Display this message and exit.
-  -b|--branch -  Branch name (default is branch equivalent to the current MLRun version).
-  -u|--user   -  Username, which determines the directory to which to copy the files - /v3io/users/<username>.
-                 Default = \$V3IO_USERNAME (if set to a non-empty string).
-  --mlrun-ver -  Determine the branch based on the MLRun version, unless -b/--branch option is specified.
-  --dry-run   -  Do not update the files, but rather show the pending changes.
-  --no-backup -  Do not backup the current demos directory."
+  -b|--branch -  Git branch name. Default: The latest release branch that
+                 matches the version of the running MLRun service.
+  -u|--user   -  Username, which determines the directory to which to copy the
+                 retrieved demo files (/v3io/users/<username>).
+                 Default: \$V3IO_USERNAME, if set to a non-empty string.
+  --mlrun-ver -  The MLRun version for which to get demos; determines the Git
+                 branch from which to get the demos, unless -b|--branch is set.
+                 Default: The version of the running MLRun service.
```

Comment:
Same as above

Reply:
Fixed — see my response to your `-b|--branch` comment — https://github.com/v3io/tutorials/pull/308#discussion_r547123857.